tools/chapel-py/scripts/generate-pyi.py

```diff
@@ -0,0 +1,202 @@
+#!/usr/bin/env python3
+#
+# Copyright 2020-2023 Hewlett Packard Enterprise Development LP
+# Copyright 2004-2019 Cray Inc.
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import chapel.core
+from typing import List, Tuple
+
+
+def get_base_header() -> str:
+    c = chapel.core.Context()
+    return c._get_pyi_file()
```

Comment:
Since we're not exporting this (or anything) from the function, I don't see why this shouldn't be prefixed with `_` as well.